### ISSUE TITLE
Data ownership

### ISSUE THREAD

**Issue by RobinIsTheBird** [FDPA/resolutions#3]
The fusiontables are still not owned by any account on the fairdistrictspa.com domain. @sarawb, does it matter? @amyprime points out that it would be disruptive to dozens of resolutions activists to change the ownership.

**Comment by sarawb** [comment]
From a long-term governance and risk perspective, FDPA-owned accounts are of course ideal. If that's not practical right now, then it's not practical! I am not in charge of deciding whether it's officially OK or not, but I would agree that we have a lot of contributors who are not tech-savvy and may struggle with this shift. Maybe we can leave it alone for now, then reassess if/when something shifts in terms of how resolutions are being handled or what the strategy is statewide.

**Comment by RobinIsTheBird** [comment]
Do we want to add prioritization labels? Perhaps the thing to do is to leave this open, remove the question label, but add a low priority label?

**Comment by amyprime** [comment]
I think I'd like to create a new gmail account (e.g. fdpa.resolutions@gmail.com) and transfer ownership to that.  That way at least a few people can have the password.  @sarawb if you have suggestions of who should have the credentials (besides me and Jamie), let me know.

**Comment by RobinIsTheBird** [comment]
Might be easier for me to contribute software if I have credentials, too.

I wonder whether the new account should be in the gmail domain or the fairdistrictspa.com domain. The latter is a google apps account, if I'm not mistaken, so someone with admin privileges could easily add an account to it. An earlier message from @sarawb implied that there already is a `resolutions` account in the fairdistrictspa.com domain, but the new one could be `fusiontables` or `resolutions.dev` or something like that. I'd be in favor of `fusiontables` because it's somewhat open ended with regard to what other data besides resolutions might be added to `fusiontables` and accessed together in a join table. An example would be some of Steve Elfelt's ideas.

We need more input from Sara.

**Comment by amyprime** [comment]
You can't transfer Google docs between domains (i.e., gmail.com to fairdistrictspa.com), even if both are Google domains.  All the "solutions" you'll find involve creating new copies of the docs, which I don't want to do.  Note that resolutions@fairdistrictspa.com does have permissions as an editor of the Fusion Tables -- I'm just talking about the actual ownership.

**Comment by RobinIsTheBird** [comment]
Right, I understand the diff between editor permissions and ownership. Yes, you'd have to copy the data.

@amyprime, what are the arguments against copying the data?

**Comment by amyprime** [comment]
Creating new merges and updating the links and stuff would be a pain, but I could work through that.  The main problem is that I have about 35 county-specific tables (views of the main table) that are maintained by local group resolutions coordinators, many of whom are not tech-savvy and would be very confused by having to switch links.  If they were spreadsheets, I could go through them, delete the content, and place a link to the new version so there's no confusion (I in fact did this when we switched from sheets to tables), but I can't do that with the way Fusion Tables works.

**Comment by RobinIsTheBird** [comment]
I see. Makes sense. I haven't seen the fusion tables as a content provider, only as a dev doing queries.

Do the resolutions coordinators have API keys or OAuth credentials that would have to change if the fusiontables ownership were transferred to an `fdpa...@gmail.com` account?

What do you think of naming it something like `fdpa.fusiontables` to leave it open ended for adding sponsorship and candidate tables, @amyprime? The reason for adding those tables to the same account would be for the sake of merges (joins), and creating overlays on the same map.

Steve Elfelt's vision is something like seeing the counties & municipalities with resolutions in the state districts for PA senators and reps who are not sponsors, and using that visual to hold the legislators accountable. I don't know enough about fusiontables yet to know if that could be done with different ownership.

**Comment by amyprime** [comment]
All the coordinators need is a Google account (which may be gmail.com or not).  It's completely transparent to them if I move the ownership to a different gmail.com account, so no issues there.

It's also perfectly possible to do merges and things across accounts, but it would probably be nice from an organizational perspective to have all the tables in one place.  Maybe call it fdpa.googledocs to be even more generic?  OTOH, it may be better to set up any new tables under the fairdistrictspa.com domain.

Michael had already put together maps overlaying resolutions with sponsorships, but I haven't been advertising them because the sponsorship data was from a one-off table, so it's not up to date:
https://amyprime.github.io/fdpa-resolutions-senate.html
https://amyprime.github.io/fdpa-resolutions-house.html

**Comment by amyprime** [comment]
@sarawb The account operations.fdpa@gmail.com seems to have ownership or edit access to a few different "official" Google docs.  Do you think it would be good to standardize on this account, rather than making a new one?  (By "official" I mean documents that I know to be used at a statewide level or distributed publicly.)

So that I don't lose this information, I'll include the results of my sleuthing here:

OWNER -- DOCUMENT
operations.fdpa@gmail.com -- Redistricting Resolution Support Form 2017 (Responses)
carolkuniholm@gmail.com -- FDPA Event Follow-up Report [operations.fdpa editor]
amy.finkbiner@gmail.com -- Facebook - Local Groups [operations.fdpa editor]
??? "Info FDPA" -- FDPA Statewide Leadership Team with Regional Groups
??? "Fair Districts PA" -- FDPA Legislative Response.xlsx
amy.finkbiner@gmail.com -- Resolutions Fair Districts PA

In addition, operations.fdpa@gmail.com currently has edit access to "FDPA Event Follow-up Report" and "Facebook - Local Groups."

**Comment by RobinIsTheBird** [comment]
@amyprime, Michael's maps look great. The senate map contains one blue district that I don't see explained in the legend. Do you know what that one is?

Is there some sort of record of sponsorship that is being kept up to date, that we could modify Michael's maps to leverage?

I agree, there should be one official owner for all long-lived statewide documents and databases (fusiontables really is a sort of database). Since there are so many issues with transferring docs across domains and so much already exists in the gmail.com domain, I think we should create a `xxx.fdpa@gmail` or reuse `operations.fdpa@gmail`. I like @amyprime's suggestion of `fdpa.googledocs`, but in reverse, as `googledocs.fdpa` to be consistent with `operations.fdpa`.

It may make sense to have different ownership for ongoing data that gets updated over time from fugitive data such as event organization and timely notifications. My guess is that a name like `operations.fdpa` would be suited to the latter. What are your thoughts, @sarawb?

An account in the `fairdistrictspa.com` domain for long lived data is a good idea, but we need to check the implications. I think the thing to do is:

1. Consolidate under a `xxx.fdpa@gmail` account first.
2. Try creating a couple documents under an account in the `fairdistrictspa.com` domain.
3. Check documents in the two accounts for interoperability, e.g. merging fusiontables, hyperlinks, cross refs between Google sheets, etc. before committing to the `fairdistrictspa.com` account.
